packages/pg-cursor/package.json

```
@@ -18,5 +18,8 @@
   "devDependencies": {
     "mocha": "^7.1.2",
     "pg": "^8.5.1"
+  },
+  "peerDependencies": {
+    "pg": ">=8.0"
```

I’m not sure if this should assume it’ll be compatible with a new major version of pg. If it turns out to be, a new release can always expand the range. So, `"pg": "^8"`?
```suggestion
    "pg": "^8"
```